pkg/errors/errorMessage.go

```diff
@@ -250,52 +250,54 @@ var (
 	GALASA_ERROR_MISSING_USER_LOGIN_ID_FLAG           = NewMessageType("GAL1155E: The loginId provided by the --login-id field cannot be an empty string.", 1155, STACK_TRACE_NOT_WANTED)
 	GALASA_ERROR_RETRIEVING_USER_LIST_FROM_API_SERVER = NewMessageType("GAL1156E: Could not get list of users from API server. Reason: '%s'."+
 		" Ensure you have allocated a personal access token and configured your client program by setting your GALASA_TOKEN as an environment variable or by storing it in your galasactl.properties file", 1156, STACK_TRACE_WANTED)
-	GALASA_ERROR_DELETE_RUN_FAILED         = NewMessageType("GAL1157E: An attempt to delete a run named '%s' failed. Cause is %s", 1157, STACK_TRACE_NOT_WANTED)
-	GALASA_ERROR_SERVER_DELETE_RUNS_FAILED = NewMessageType("GAL1158E: An attempt to delete a run named '%s' failed. Sending the delete request to the Galasa service failed. Cause is %v", 1158, STACK_TRACE_NOT_WANTED)
-	GALASA_ERROR_INVALID_LOGIN_ID          = NewMessageType("GAL1165E: '%s' is not supported as a valid login ID. Login ID should not contain spaces.", 1165, STACK_TRACE_NOT_WANTED)
-	GALASA_ERROR_INVALID_USER_FLAG_VALUE   = NewMessageType("GAL1166E: The loginId provided by the --user field cannot be an empty string.", 1166, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_DELETE_RUN_FAILED            = NewMessageType("GAL1157E: An attempt to delete a run named '%s' failed. Cause is %s", 1157, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_SERVER_DELETE_RUNS_FAILED    = NewMessageType("GAL1158E: An attempt to delete a run named '%s' failed. Sending the delete request to the Galasa service failed. Cause is %v", 1158, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_INVALID_LOGIN_ID             = NewMessageType("GAL1165E: '%s' is not supported as a valid login ID. Login ID should not contain spaces.", 1165, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_INVALID_USER_FLAG_VALUE      = NewMessageType("GAL1166E: The loginId provided by the --user field cannot be an empty string.", 1166, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_FAILED_TO_DELETE_USER        = NewMessageType("GAL1167E: Failed to delete user from database by user number.", 1167, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_SERVER_DELETE_USER_NOT_FOUND = NewMessageType("GAL1168E: The user could not be deleted because it was not found by the Galasa service. Try listing runs using 'galasactl runs get' to identify the one you wish to delete", 1168, STACK_TRACE_NOT_WANTED)
```

Comment:
GAL1165-68 already exist - placing these new errors in between the existing `runs delete` errors also makes it a bit less organised (i.e. we currently have GAL1157-64, but the PR adds 64-68 after 58)

Reply:
Also for GALASA_ERROR_INVALID_USER_FLAG_VALUE, is `--user` correct? Isn't it `--login-id` now?

Reply:
GALASA_ERROR_SERVER_DELETE_USER_NOT_FOUND mentions:
```
Try listing runs using 'galasactl runs get' to identify the one you wish to delete
```
which isn't right!